test/functional/treesitter/parser_spec.lua

```diff
@@ -572,28 +600,5 @@ int x = INT_MAX;
         eq(result, "value")
       end)
     end)
-
-    describe("when setting for a capture match", function()
```

Comment:
Is this no longer possible to do, is that why the test is removed?

Reply:
The old implementation had the concept of storing data pure node in the metadata strictly for the way offset was storing data. This was removed since we only need the metadata for the match instead of on a capture basis. So this is no longer used and was removed.